docs/quickstarts/twitter.rst

```diff
@@ -24,7 +24,7 @@ Code
     )
     app.register_blueprint(blueprint, url_prefix="/login")
 
-    @app.route("/")
+    @app.route("/twitter/login")
```

Comment:
Why are you changing this URL?

Reply:
@singingwolfboy sorry I was confused here - I'll change it back - I thought one would need a dedicated route for twitter auth and that the "/" route would be used to return your index page.